docs/configuration/alertmanager-integration/victoria-metrics.rst

```diff
@@ -8,26 +8,49 @@ You will need to configure two integrations: one to send alerts to Robusta and a
 Send Alerts to Robusta
 ============================
 
-To configure, edit AlertManager's configuration:
+Add the following to your Victoria Metrics Alertmanager configuration (e.g., Helm values file or VMAlertmanagerConfig CRD):
+
+.. code-block:: yaml
+
+    receivers:
+        - name: 'robusta'
+        webhook_configs:
+            - url: 'http://<ROBUSTA-HELM-RELEASE-NAME>-runner.<NAMESPACE>.svc.cluster.local/api/alerts'
+            send_resolved: true # (3)
+
+    route: # (1)
+        routes:
+        - receiver: 'robusta'
+            group_by: [ '...' ]
+            group_wait: 1s
+            group_interval: 1s
+            matchers:
+            - severity =~ ".*"
+            repeat_interval: 4h
+            continue: true # (2)
+
+.. code-annotations::
+    1. Put Robusta's route as the first route, to guarantee it receives alerts. If you can't do so, you must guarantee all previous routes set ``continue: true`` set.
+    2. Keep sending alerts to receivers defined after Robusta.
+    3. Important, so Robusta knows when alerts are resolved.
 
-.. include:: ./_alertmanager-config.rst
 
 .. include:: ./_testing_integration.rst
 
 Configure Metric Querying
 ====================================
 
-Metrics querying lets Robusta pull metrics and create silences.
+Robusta can query metrics and create silences using Victoria Metrics. If both are in the same Kubernetes cluster, Robusta can auto-detect the Victoria Metrics service. To verify, go to the "Apps" tab in Robusta, select an application, and check for usage graphs.
 
-Add the following to ``generated_values.yaml`` and :ref:`update Robusta <Simple Upgrade>`.
+If auto-detection fails or if Robusta and Victoria Metrics are in different clusters, add the following to ``generated_values.yaml`` and :ref:`update Robusta <Simple Upgrade>`.
 
 .. code-block:: yaml
 
     globalConfig: # this line should already exist
         # add the lines below
-        alertmanager_url: "http://ALERT_MANAGER_SERVICE_NAME.NAMESPACE.svc.cluster.local:9093"
+        alertmanager_url: "http://<VM_ALERT_MANAGER_SERVICE_NAME>.<NAMESPACE>.svc.cluster.local:9093" # Example:"http://vmalertmanager-victoria-metrics-vm.default.svc.cluster.local:9093/"        
         grafana_url: ""
```

Comment:
I would remove `grafana_url` from here - it doesn't add anything in this context